docs/intro/install.rst

```diff
@@ -1,245 +1,56 @@
-development headers. On OS X this is typically provided by Apple’s Xcode
-development tools. To install the Xcode command line tools open a terminal
-window and run::
-
-    xcode-select --install
-
-There's a `known issue <https://github.com/pypa/pip/issues/2468>`_ that
-prevents ``pip`` from updating system packages. This has to be addressed to
-successfully install Scrapy and its dependencies. Here are some proposed
-solutions:
-
-* *(Recommended)* **Don't** use system python, install a new, updated version
-  that doesn't conflict with the rest of your system. Here's how to do it using
-  the `homebrew`_ package manager:
-
-  * Install `homebrew`_ following the instructions in https://brew.sh/
-
-  * Update your ``PATH`` variable to state that homebrew packages should be
-    used before system packages (Change ``.bashrc`` to ``.zshrc`` accordantly
-    if you're using `zsh`_ as default shell)::
-
-      echo "export PATH=/usr/local/bin:/usr/local/sbin:$PATH" >> ~/.bashrc
-
-  * Reload ``.bashrc`` to ensure the changes have taken place::
-
-      source ~/.bashrc
-
-  * Install python::
-
-      brew install python
-
-  * Latest versions of python have ``pip`` bundled with them so you won't need
-    to install it separately. If this is not the case, upgrade python::
-
-      brew update; brew upgrade python
-
-* *(Optional)* Install Scrapy inside an isolated python environment.
-
-  This method is a workaround for the above OS X issue, but it's an overall
-  good practice for managing dependencies and can complement the first method.
-
-  `virtualenv`_ is a tool you can use to create virtual environments in python.
-  We recommended reading a tutorial like
-  http://docs.python-guide.org/en/latest/dev/virtualenvs/ to get started.
-
-After any of these workarounds you should be able to install Scrapy::
-
-  pip install Scrapy
-
-
-PyPy
-----
-
-We recommend using the latest PyPy version. The version tested is 5.9.0.
-For PyPy3, only Linux installation was tested.
-
-Most Scrapy dependencides now have binary wheels for CPython, but not for PyPy.
-This means that these dependecies will be built during installation.
-On OS X, you are likely to face an issue with building Cryptography dependency,
-solution to this problem is described
-`here <https://github.com/pyca/cryptography/issues/2692#issuecomment-272773481>`_,
-that is to ``brew install openssl`` and then export the flags that this command
-recommends (only needed when installing Scrapy). Installing on Linux has no special
-issues besides installing build dependencies.
-Installing Scrapy with PyPy on Windows is not tested.
-
-You can check that Scrapy is installed correctly by running ``scrapy bench``.
-If this command gives errors such as
-``TypeError: ... got 2 unexpected keyword arguments``, this means
-that setuptools was unable to pick up one PyPy-specific dependency.
-To fix this issue, run ``pip install 'PyPyDispatcher>=2.1.0'``.
```

Comment:
Unfortunately this kind of problems is not covered anywhere as well. Is there a better place for such notes?